src/features/landing/components/Header/HeaderMobileLink/HeaderMobileLink.tsx

```diff
@@ -0,0 +1,50 @@
+import { Grid, Typography } from '@mui/material';
+import { LinkProps } from 'next/link';
+
+import CustomLink from '@/components/ui/CustomLink/CustomLink';
+
+import ArrowDownIcon from '../../Icons/ArrowDownIcon/ArrowDownIcon';
+
+interface IHeaderMobileLinkProps extends LinkProps {
+  href: string;
+  linkNameText: string;
+}
+
+const headerMobileLinkStyle: React.CSSProperties = {
```

Comment:
why you change your approach about object with styles? in all code base must be one approach about some cases, for example if you use object with styles, use only it and don't change it